### PR TITLE
xmr: get_address - pagination, integrated address

### PR DESCRIPTION
**Pull request by ph4r05**
- pagination added to show_address
- integrated address supported
- QR code fixed (prefix with monero: as GUI does)

Relates to (PR to Monero will follow):
- https://github.com/monero-project/monero-gui/issues/2184
- https://github.com/monero-project/monero/pull/5594

Btw the trezor-common repo is synced automatically?

Regarding the UI: I've added the get_address with scrolling to monero in order to avoid shared code modification (import changes -> memory increase). Feel free to revamp the UI. This is the working prototype for get_address to work.

Thanks!

### CODE REVIEW

File: core/src/apps/monero/get_address.py [hunk 1]
```diff
@@ -11,13 +15,37 @@
     )
 
     creds = misc.get_creds(keychain, msg.address_n, msg.network_type)
+    addr = creds.address
+
+    if msg.payment_id:
+        utils.ensure(len(msg.payment_id) == 8, "Invalid payment ID length")
```
Comment (tsusanka) [inline core/src/apps/monero/get_address.py]: I believe 
```
if len(msg.payment_id) != 8:
  raise ValueError("Invalid payment ID length")
```
is way more readable.

File: core/src/apps/monero/layout/confirms.py [hunk 2]
```diff
@@ -194,3 +194,30 @@ def on_render(self):
...
+        text.mono(*lines)
+        pages.append(text)
+
+    return await confirm(
+        ctx,
+        Paginated(pages),
+        code=ButtonRequestType.Address,
+        cancel="QR",
+        cancel_style=ButtonDefault,
+    )
```
Comment (tsusanka) [inline core/src/apps/monero/layout/confirms.py]: @jpochyla could you look on this file? Just to make sure this is a correct use of the new UI

Comment (jpochyla) [reply core/src/apps/monero/layout/confirms.py]: Yes looks alright!

Comment (tsusanka) [reply core/src/apps/monero/layout/confirms.py]: ACK from him

**Comment by tsusanka** [comment]
@matejcik should update trezor-common repo this tonight

**Pull request merged**